src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      count: action.payload
```

Comment:
You want to make sure your state is immutable, spreading in the old state object in this way will assure that.
```suggestion
      .... state, count: action.payload
```